packages/stores/types.ts

```diff
@@ -180,6 +213,61 @@ type TClientStore = {
     logout: () => Promise<LogOutResponse>;
     should_allow_authentication: boolean;
     is_crypto: boolean;
+    isEligibleForMoreDemoMt5Svg: (market_type: 'synthetic' | 'financial') => boolean;
+    isEligibleForMoreRealMt5: (market_type: 'synthetic' | 'financial') => boolean;
+    fetchResidenceList?: () => void;
+    account_settings: GetSettings & {
+        upload_file?: string;
+        poi_state?: string;
+    };
+    residence_list: ResidenceList;
+    is_high_risk: boolean;
+    should_restrict_bvi_account_creation: boolean;
+    should_restrict_vanuatu_account_creation: boolean;
+    trading_platform_available_accounts: {
+        market_type: 'financial' | 'gaming';
+        name: string;
+        requirements: {
+            after_first_deposit: {
+                financial_assessment: string[];
+            };
+            compliance: {
+                mt5: string[];
+                tax_information: string[];
+            };
+            signup: string[];
+        };
+        shortcode: 'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest';
+        sub_account_type: string;
+    }[];
+    updateMT5Status: () => void;
+    fetchAccountSettings: () => void;
+    setAccountSettings: (get_settings_response: GetSettings) => void;
+    upgradeable_landing_companies: unknown[];
+    is_populating_mt5_account_list: boolean;
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    landing_companies: Record<string, any>;
+    getChangeableFields: () => string[];
+    landing_company: LandingCompany;
+    isAccountOfTypeDisabled: (account: Record<string, DetailsOfEachMT5Loginid>) => boolean;
+    is_mt5_allowed: boolean;
+    mt5_disabled_signup_types: {
+        real: boolean;
+        demo: boolean;
+    };
+    dxtrade_disabled_signup_types: {
+        real: boolean;
+        demo: boolean;
+    };
+    dxtrade_accounts_list_error: null;
```

Comment:
@amir-deriv This one is weird 😅 If it's always `null` then its a dead code I guess 👀

Reply:
I guess I copied it from somewhere in the existing code 😄 , let me check this

Reply:
yes, you are right @farzin-deriv , the default value of `dxtrade_accounts_list_error` is `null` and it's not getting changed in the state from anywhere

Reply:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/129206554/233426687-8dbf7410-0562-4e8a-bffd-7051e0c14e8d.png">